inlang/source-code/sdk/src/loadProject.test.ts

```diff
@@ -197,6 +197,42 @@ describe("initialization", () => {
 		expect(result.data).toBeDefined()
 	})
 
+	it("should generate projectId on missing projectid in local repo", async () => {
+		const repo = await mockRepo({ openLocal: true })
```

Comment:
I would NOT expect an argument to `mockRepo`. @janfjohannes Please change the API to

```ts
const repo = await virtualRepository()

const repo = await virtualRepository({ fromSnapshot: "" })

const repo = await virtualRepository({ fromFilesystem: ""})
```